src/utils/gameStateManager.ts

```
@@ -0,0 +1,1103 @@
+      // Only valid during TESTING_PHASE
+      if (state.phase !== 'TESTING_PHASE') {
+        console.warn(
+          'Invalid action: Can only add test results during TESTING_PHASE'
+        );
+        return state;
+      }
+
+      const newScore = action.result.isCorrect
+        ? state.score + 1
+        : state.score;
+      const newCritterState: CritterState = action.result.isCorrect
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)